service/build.gradle

```diff
@@ -102,13 +104,17 @@ dependencies {
     implementation group: 'com.azure', name: 'azure-identity', version: '1.3.5'
     implementation group: 'com.azure', name: 'azure-core', version: '1.19.0'
     implementation group: 'com.azure.resourcemanager', name: 'azure-resourcemanager', version: '2.8.0'
+    implementation group: 'com.azure.resourcemanager', name: 'azure-resourcemanager-compute', version: '2.7.0'
```

Comment:
Does this need to be here? I think CRL should pull in azure dependencies (can probably remove this whole block).

Reply:
There's some initial code in WSM that use on this, written prior to CRL usage in WSM. some of which Dan's tests depend on. I can remove my additions at this point though.